doc/whats-new.rst

```diff
@@ -49,10 +49,11 @@ New Features
 
 Bug fixes
 ~~~~~~~~~
+- Variables which are chunked using dask only along some dimensions can be chunked while storing with zarr along previously
+  unchunked dimensions.
```

Comment:
Please feel free to take credit by adding your name and github id here if you would like to.

Reply:
Thanks for the hint. I've added my name and the pull request id.